stories/Documentation/ComponentContribution.mdx

```diff
@@ -0,0 +1,464 @@
+import { Meta } from '@storybook/blocks';
+
+<Meta
+  title="Getting started/Component Contribution Guide"
+  parameters={{
+    viewMode: "docs",
+    previewTabs: {
+      canvas: { hidden: true }
+    }
+  }}
+/>
+
+# Component Contribution Guide
+
+This guide explains how to contribute new components to the Mangrove library and outlines the standards and best practices to follow.
+
+## Component Development Workflow
+
+### 1. Planning Phase
+
+Before starting development:
+
+- Check if a similar component already exists to avoid duplication
+- Define the component's purpose, functionality, and variants
+- Identify the props/parameters the component will accept
+- Consider accessibility requirements from the beginning
+- Plan responsive behaviors for different screen sizes
+
+### 2. Component Structure
```

Comment:
I had created a TypeScript example component, maybe we want to do this for a normal component?

https://github.com/unisdr/undrr-mangrove/tree/main/stories/Components/TypeScriptExampleComponent

Reply:
The role for TS in mangrove is still an open discussion point (it's on the other side of the tech team to advocate for that). What I was getting at here was I think it would be nice to have a normal React JSX example component as a quick star.